src/ocnicepost.fd/ocnicepost.F90

```diff
@@ -0,0 +1,577 @@
+program ocnicepost
+
+  use utils_mod, only : debug, logunit, getfield, packarrays, remap, dumpnc
+  use outputvars
+  use netcdf
+
+  implicit none
+
+  character(len= 20) :: fnml, ftype
+  character(len=120) :: wgtsdir
+  character(len=120) :: input_file, wgtsfile, output_file
+
+  ! source grid, tripole 1/4 deg, 40 vertical levels
+  integer, parameter :: nxt = 1440, nyt = 1080, nlevs = 40
```

Comment:
Is the input file required to be 1/4 degree?  Will this program work for MOM6/CICE6 output at any other resolution e.g. 5 degree?

Reply:
The required weights are generated for 1/4 tripole->(rect 1/4deg, 1/2deg, 1deg), the 1/2 tripole -> (rect 1/2deg, 1deg) and 1deg tripole>rect 1deg. This initial code was intended as a demonstration of how the original NCL scripts could be rewritten in fortran. The NCL scripts could do any of the three tripole resolutions.

The other resolutions can all be added as variations to the fortran version. As noted above however, the 5deg ocean output was never intended as being used for anything other than plumbing. Are you suggesting that the 5deg tripole output will be used for some purpose? Or do you want it for plumbing in this case also?

The destination grid resolutions themselves are fixed. That is, the weights are pre-generated and work only for a rectilinear 1/4deg of 1440x721, for example The code is not expected to operate between completely arbitrary destination and sources.

Reply:
We discussed this offline.
We will put in place checks to ensure that the target grid resolution does not exceed the input grid resolution; i.e. a 5-deg tripolar input will only produce 5-deg rectilinear output.  1/4-deg tripolar input can produce 1/4, 1/2, 1, 5-deg rectillinear outputs.  This will be namelist configurable rather than looping over in the code.